llmfoundry/command_utils/train.py

```diff
@@ -311,18 +311,19 @@ def train(cfg: DictConfig) -> Trainer:
     eval_gauntlet_config = train_cfg.eval_gauntlet or train_cfg.eval_gauntlet_str
 
     # Optional parameters will be set to default values if not specified.
-    env_run_name: Optional[str] = os.environ.get('RUN_NAME', None)
-    run_name: str = (
-        train_cfg.run_name if train_cfg.run_name else env_run_name
-    ) or 'llm'
+    run_name: Optional[
+        str] = train_cfg.run_name if train_cfg.run_name else os.environ.get(
+            'RUN_NAME',
+            None,
+        )
     is_state_dict_sharded: bool = (
         fsdp_config.get('state_dict_type', 'full') == 'sharded'
     ) if fsdp_config else False
     save_latest_filename: str = train_cfg.save_latest_filename if train_cfg.save_latest_filename else 'latest-sharded-rank{rank}' if is_state_dict_sharded else 'latest-rank{rank}.pt'
     save_filename: str = train_cfg.save_filename if train_cfg.save_filename else 'ep{epoch}-ba{batch}-rank{rank}.pt'
 
     # Enable autoresume from model checkpoints if possible
-    is_user_set_run_name: bool = train_cfg.run_name is not None or env_run_name is not None
+    is_user_set_run_name: bool = train_cfg.run_name is not None or run_name is not None
     autoresume_default: bool = False
     if is_user_set_run_name and \
         train_cfg.save_folder is not None \
```

Comment:
```suggestion
    autoresume_default: bool = False
    if run_name is not None and \
        train_cfg.save_folder is not None \
```
only used in one spot and also train_cfg is subset of run_name